### PR TITLE
hw11: tips and tricks

### PR DESCRIPTION
**Pull request by kamiour**
- used SCSS modules to encapsulate styles
- integrated Storybook
- Lighthouse performance: 98

### CODE REVIEW

File: src/components/NextImageCustom/NextImageCustom.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ export const NextImageCustom = ({ alt, ...props }) => {
 
   return (
     <Image
+      unoptimized // to make it work in Storybook
```
Comment (Twixs) [inline src/components/NextImageCustom/NextImageCustom.tsx]: and what about production?))
storybook is not the priority and `unoptimized` value could be controlled via env variables
https://nextjs.org/docs/basic-features/environment-variables

**Review by Twixs — APPROVED** [review]
In general good. Mostly in stories we describe UI state of some components of our app to show how they look like and their variations based on params. In your case you also described in stories forms/form state which can be tested with unit tests instead of it you could have stories for how modal looks like (w/ or w/o title), how it appears etc.

File: src/components/NextImageCustom/NextImageCustom.tsx (hunk 1 above)
Comment (mykhailo-kazakov) [reply src/components/NextImageCustom/NextImageCustom.tsx]: Yes, I forgot about production... I fixed it using `process.env.NODE_ENV`

Comment (kamiour) [reply src/components/NextImageCustom/NextImageCustom.tsx]: Yes, I forgot about production... I fixed it using `process.env.NODE_ENV`, thank you!

**Review by Twixs — APPROVED** [review]

**Pull request merged**